### PR TITLE
feat: bigint uses decimal over hex encoding

### LINKED ISSUE #435: REST API: BigInt Conversion of Hex-encoded numbers to String Literals/Decimals

### PR DESCRIPTION
**Pull request by AuHau**
Closes #435

### CODE REVIEW

File: codex/utils/stintutils.nim [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import pkg/stint
+
+func fromString*(T: typedesc[StUint|StInt], s: string): T {.inline.} =
```
Comment (markspanbroek) [inline codex/utils/stintutils.nim:3]: I would suggest to call this `fromDecimals()`, to be more explicit. And it matches the `fromHex()` naming convention.

**Review by markspanbroek — APPROVED** [review]
Nice :+1:

File: codex/utils/stintutils.nim (hunk 1 above)
Comment (AuHau) [reply codex/utils/stintutils.nim:3]: Good idea, thanks! 👍

**Comment by mratsim** [comment]
@jangko is revamping stint. Currently on serialization https://github.com/status-im/nim-stint/pull/132, that makes sense to be upstream.

I would name if `fromDecimal` instead of `fromDecimals` though since it's decimal representation.

**Comment by AuHau** [comment]
>@jangko is revamping stint. Currently on serialization https://github.com/status-im/nim-stint/pull/132, that makes sense to be upstream.

Sorry, not sure what you mean here 😅 Do you want to have the `fromDecimal` directly in `stint`?

>I would name if fromDecimal instead of fromDecimals though since it's decimal representation or a decimal number.
 
Yeah that is actually what I used, so all good 👍

**Comment by jangko** [comment]
okay, will add `fromDecimal` to stint

**Pull request merged**